src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -380,7 +372,7 @@ def link_content(text, css_class, icon)
     if css_class && css_class.include?('nav-link')
       capture do
         concat(tag.i('', class: "fas #{icon}"))
-        concat(tag.div(text, class: feature_enabled?(:responsive_ux) ? '' : 'small'))
+        concat(tag.div(text))
```

Comment:
Note for reviewers: This helper `link_content` is used for the `Sign Up` and `Log In` links. How they are displayed shouldn't have changed at all.